WeakAuras/Transmission.lua

```diff
@@ -934,6 +987,9 @@ function WeakAuras.ShowDisplayTooltip(data, children, icon, icons, import, compr
             data.parent = nil;
             findUnusedId(data);
             importData(data);
+            if matchedData then
+              data.uid = nil
```

Comment:
Can you move those two lines above the importData. It makes it cleared that the uid is cleared in the imported data.